src/main/java/racingcar/exception/InputException.java

```diff
@@ -0,0 +1,16 @@
+package racingcar.exception;
+
+public class InputException extends RuntimeException{
```

Comment:
아래와 같은 정렬이 안되는 부분이 간혹 보였었는데요.
IDE에서 제공해주는 정렬기능을 구현하시고 나서 나중에 적극 활용하시면 좋을 것 같네요.
```suggestion
public class InputException extends RuntimeException {
```

> InputException을 커스텀 예외로 분리하였는데(nextInt, nextLine의 예외가 다른데 하나로 처리하는게 나을 것 같다는 생각이었습니다), 나머지는 주로 IllegalArgumentException이어서 그대로 두었는데요(예외 의미가 이미 명확한 것 같아서), 이런것들도 커스텀 예외 처리를 하고 메시지를 좀 더 간소화 시키는게 나을까요?

커스텀 예외가 필요한 시점에서는 분리하는게 저는 더 적절하다고 생각합니다.
다만, 재사용성도 고려해서 구현하시는게 좋을 것 같아요. 😄

Reply:
아아 ㅜ_ㅜ 통일성,가독성 있는 코드를 위해서 정렬 신경쓰도록 하겠습니다! 🤭

넵!! 커스텀 예외 생성과 관련해서는 조금 더 스스로 생각해보겠습니다! 구현하면서 분리가 필요한 시점이나 재사용될 가능성에 대해서 계속 고민해봐야겠네요 ㅎㅎ 🧐